.ci.yaml

```diff
@@ -4139,3 +4097,1247 @@ targets:
     properties:
       tags: >
           ["framework", "hostonly", "shard"]
+
+# Staging builds are specially constructed tasks.
+#
+# Names should begin with `Staging_build_` prefix.
+# Task should include presubmit: false
+# Task should inherit ignore_flakiness: true
```

Comment:
https://github.com/flutter/cocoon/pull/1828 is relying on target level property. So the platform property defined in this PR will not work.
We need to refactor the logic in https://github.com/flutter/cocoon/pull/1828 to incorporate https://github.com/flutter/cocoon/pull/1837.

Reply:
Added as target level property.